client/my-sites/earn/controller.ts

```diff
@@ -0,0 +1,29 @@
+import page from 'page';
+import { createElement, FC } from 'react';
+import Main from './main';
+import type { Context as PageJSContext } from 'page';
```

Comment:
For imports, changes are to add imports for FC from React and for PageJSContext.